src/Core/ContentManager/Permalink/PermalinkGenerator.php

```diff
@@ -226,7 +226,7 @@ private function getTitleSlugified(ItemInterface $item)
         $preservePathTitle = $this->getPreservePathTitleAttribute($item);
 
         if ($preservePathTitle === true && isset($attributes['title_path']) === true) {
-            return (new StringWrapper($attributes['title_path']))->slug();
+            return $attributes['title_path'];
```

Comment:
Right, well then maybe use urlencode() instead of slug(), so e.g.
spaces are turned to %20 and such, to try and ensure good/valid URLs,
but nothing else.